THIRD-PARTY-NOTICES

```diff
@@ -13,93 +13,42 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 This project includes:
```

Comment:
Update to parent pom, fixes the generated notice, to ONLY include compile dependencies